gen/gen.c

```diff
@@ -2905,9 +2905,8 @@ rfc2544_test(void)
 		logging("start rfc2544 test mode. trial-duration is %d sec and interval is %d sec. warming up...",
 		    opt_rfc2544_trial_duration, opt_rfc2544_interval);
 
-		/* disable transmit */
-		transmit_set(0, 0);
-		transmit_set(1, 1);
+		transmit_set(0, 0); /* if0: disable transmit */
```

Comment:
I think it's better to use the explicit word `interface[0]` instead of an ambiguous word `if0`.

Reply:
done